api/utils/models.py

```diff
@@ -1,3 +1,39 @@
+import datetime
+import jwt
+
+from django.template.loader import render_to_string
+from django.core.mail import send_mail
 from django.db import models
+from django.utils import timezone
+from django.core.signing import TimestampSigner, b64_encode,b64_decode, BadSignature, SignatureExpired,force_bytes
+from django.core.mail import EmailMessage
+from django.conf import settings 
+
+from ext_news.models import News
+
+class Mail(models.Model):
```

Comment:
do we need extra class for simple mail sender?

Reply:
already done